documentation/user_docs/docs/user_guide/Interfacing_with_other_programs.rst

```diff
@@ -0,0 +1,869 @@
+###############################
+Interfacing with other programs
+###############################
+
+Horace is designed to be able to call other programs to calculate models for simulating and fitting data.
+
+This document describes two interfaces with other `PACE <https://www.isis.stfc.ac.uk/Pages/Proper-analysis-of-coherent-excitations.aspx>`__ projects:
+
+- `Euphonic <https://euphonic.readthedocs.io/>`__
+
+- `SpinW <https://spinw.org/>`__
+
+as well as a generic way of interfacing with codes you might have written.
```

Comment:
Maybe "any codes you have written". I might have written a lot of codes - I have very good intentions - but few have made it to the light

Reply:
Changed to "... with modelling codes you may want to use with Horace."

And added a comment to look up Euphonic / SpinW for how to use Brille with them (this documentation does not actually exist yet, because the Euphonic-Brille interface inside Euphonic is not written and Brille functionality with magnons is buggy and so has not been documented!)